kyuubi-server/web-ui/.gitignore

```diff
@@ -0,0 +1,38 @@
+ # Licensed to the Apache Software Foundation (ASF) under one or more
+ # contributor license agreements.  See the NOTICE file distributed with
+ # this work for additional information regarding copyright ownership.
+ # The ASF licenses this file to You under the Apache License, Version 2.0
+ # (the "License"); you may not use this file except in compliance with
+ # the License.  You may obtain a copy of the License at
+ #
+ #     http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+
+.DS_Store
+node_modules
+/dist
+/coverage
+
+# local env files
+.env.local
+.env.*.local
+
+# Log files
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
```

Comment:
could we merge them into `*.log*` ?, BTW, why the later `*` is required?

Reply:
:) didn't look into it, it's generated by vite automatically, just guess it's the rules of npm/yarn/...'s log file name, but I know here is required,  as someone may use node 10 to install our project, it is possible to generate logs for users to check errors, but these logs do not need to be uploaded to github

Reply:
thanks for explanation